php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -1652,7 +1654,6 @@ public ExpressionTree POSTFIX_EXPRESSION() {
           MEMBER_EXPRESSION(),
           NEW_EXPRESSION(),
           EXIT_EXPRESSION(),
-          LIST_EXPRESSION_ASSIGNMENT(),
```

Comment:
I moved this LIST_EXPRESSION_ASSIGNMENT from POSTFIX_EXPRESSION to ASSIGNMENT_EXPRESSION because I see not good reason to have an assignment that is not in ASSIGNMENT_EXPRESSION.